Modules/System Tests/Cryptography Management/app.json

```diff
@@ -29,6 +29,18 @@
                              "publisher":  "Microsoft",
                              "version":  "18.3.0.0"
                          },
+                         {
+                            "id": "5d86850b-0d76-4eca-bd7b-951ad998e997",
+                            "name": "Tests-TestLibraries",
```

Comment:
Unfortunately, it is not possible to reference "Tests-TestLibraries", so LibraryRandom should be replaced with something else here.

Reply:
Can you suggest what other library I can use for randomization? Should implement my own procedure for it or just hardcode some text without randomization?

Reply:
I've replaced that with "Any" codeunit. 
By the way, why it is not possible to reference "Tests-TestLibraries" is this application marked for removal, so we should not use it in our projects?

Reply:
"Test-TestLibraries" is part of BaseApp's tests. The System Application modules & its tests should not have any dependency to BaseApp or BaseApp's test libraries.

Basically BaseApp & its Tests/TestLibraries can depend on System Application etc but not the otherway around. :)

Reply:
Okay, I see. Thanks for explanation!